signin.js

```diff
@@ -0,0 +1,72 @@
+window.onload = function(){
+
+    const inputList = document.querySelectorAll('.sign-input');
+
+    const checkEmail = function(e){
+        const value = e.target.value;
+        value.length === 0 ? noEnterdEmail() : EnterdEmail(value);
```

Comment:
삼항연산자를 사용하셔서 input 값이 비어있을 때와 아닐때를 잘 구분하셨네요!

여기서 하나 추가적인 말씀을 드리자면 빈 문자열 ('') 같은 경우
위와 같은 삼항연산자에서 falsy 하다고 평가가 되기 때문에 false와 같은 동작을 하게 됩니다!

```suggestion
        value ? noEnterdEmail() : EnterdEmail(value);
```

그래서 이와 같이 작성해도 동작이 됩니다!